deepreg/dataset/loader/interface.py

```diff
@@ -91,12 +91,12 @@ def get_dataset_and_preprocess(
         data_augmentation: Optional[Union[List, Dict]] = None,
     ) -> tf.data.Dataset:
         """
-        :param training: bool, indicating if it's training or not
-        :param batch_size: int, size of mini batch
-        :param repeat: bool, indicating if we need to repeat the dataset
-        :param shuffle_buffer_num_batch: int, when shuffling,
+        :param training: indicating if it's training or not
+        :param batch_size: total number of samples consumed per step.
```

Comment:
i'd be explicit to add "... per step, over all GPUs"

Reply:
added